src/app/auth/login/login.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component} from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { UsuarioService } from 'src/app/services/usuario.service';
+import Swal from 'sweetalert2';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.css']
+})
+export class LoginComponent {
+  loginForm:FormGroup;
+  formSubmited:boolean = false;
+  public auth2: any;
+  loginIncorrecto:boolean = false
+
+
+  constructor(private router: Router,
+    private fb: FormBuilder,
+    private usuarioService:UsuarioService) { 
+      this.loginForm = this.fb.group({
+        email:['', [Validators.required]],
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Frontend/pullRequest?prid=7127387)
```suggestion
        email:["", [Validators.required]],
```